python/convert-from-gadget-1.py

```diff
@@ -15,6 +17,19 @@ def main(ns):
     cat = Gadget1Catalog(ns.source, ptype=1)
 
     attrs = cat.attrs.copy()
+    
+    new_cat = cat.attrs.copy()
```

Comment:
I think there is no need to create a new catalog. We always explicitly tell cat.save where to save -- thus there is no 'accidental' overwrite.